guide.qmd

```diff
@@ -268,8 +228,28 @@ guide_table(
 
 Life cycle management is simplified by reducing dependencies, and should comprise a central code repository.
 
+### Dependency Management
```

Comment:
I would suggest to just add another item, instead of creating another subsection level

Reply:
Updated